src/adminPage/adminPage.component.tsx

```diff
@@ -59,7 +59,10 @@ const AdminPage = (props: AdminPageProps): ReactElement => {
       location.pathname.startsWith(adminRoutes[key])
     ) ??
       (props.adminPageDefaultTab &&
-      adminRoutes.hasOwnProperty(props.adminPageDefaultTab)
+      Object.prototype.hasOwnProperty.call(
```

Comment:
I don't like this call signature. Can we switch to using `Object.hasOwn` instead? Browser compatibility seems good since we're dropping IE with this PR anyway.

Reply:
Yeah me neither, thanks for that 😄